src/styles/utils/reset.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and paddings to all elements, reset only for elements that you actually need